app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from errors import (NotVaccinatedError,
+                    OutdatedVaccineError,
+                    NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()
```

Comment:
Consider providing a descriptive error message when raising `NotVaccinatedError`. According to the checklist, it's good practice to include a message for clarity.